ios/Helium/Info.plist

```diff
@@ -72,7 +70,7 @@
 	<string>LaunchScreen</string>
 	<key>UIRequiredDeviceCapabilities</key>
 	<array>
-		<string>armv7</string>
+		<string>arm64</string>
```

Comment:
What does this do?

Reply:
Unclear haha, just saw it in the changelist